src/main/main.ts

```diff
@@ -81,6 +85,7 @@ const standaloneExecutableArgs = [
       (trustedAppProtocolVersionSigner) =>
         `-T=${trustedAppProtocolVersionSigner}`
     ),
+  "--no-miner",
```

Comment:
We may need to set this option as default on the headless side🤔 ...

Reply:
I opened the bump pull request, https://github.com/planetarium/NineChronicles/pull/95. After it was merged, I will update this pull request 🙏🏻